test/jdk/java/util/concurrent/tck/JSR166TestCase.java

```diff
@@ -35,31 +35,43 @@
  */
 
 /*
- * @test
+ * @test id=default
  * @summary JSR-166 tck tests, in a number of variations.
  *          The first is the conformance testing variant,
  *          while others also test implementation details.
  * @build *
  * @modules java.management
  * @run junit/othervm/timeout=1000 JSR166TestCase
  * @run junit/othervm/timeout=1000 -Djava.security.manager=allow JSR166TestCase
```

Comment:
security manager testing is relatively less important.   I would move this to a lower tier, while moving a test with -Djsr166.testImplementationDetails=true into tier1

Reply:
I still think the test with -Djava.security.manager=allow is relatively low value, so would also move it into its own @test id=security-manager